picky/src/x509/pkcs7/authenticode.rs

```diff
@@ -1653,6 +1653,7 @@ mod test {
     }
 
     #[cfg(feature = "ctl_http_fetch")]
+    #[ignore = "temporarily disabled for CI issues,which is under investigation"]
```

Comment:
issue: These changes seem duplicated. I think it was supposed to be part of this PR: https://github.com/Devolutions/picky-rs/pull/256

Reply:
looks like it's gone